src/webgl/p5.Shader.js

```diff
@@ -548,6 +548,12 @@ p5.Shader = class {
           uniform.texture.src._animateGif(this._renderer._pInst);
         }
         break;
+      case gl.SAMPLER_CUBE:
```

Comment:
Is this block the same as the one above? If so, we could do:
```js
case gl.SAMPLER_2D:
case gl.SAMPLER_CUBE:
        gl.activeTexture(gl.TEXTURE0 + uniform.samplerIndex);
        uniform.texture =
          data instanceof p5.Texture ? data : this._renderer.getTexture(data);
        // ...
```
...to handle both cases with the same code